libs/base/ui/action/dropdown-item/dropdown-item.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { Icons } from '@spryker-oryx/ui/icon';
+import { hydrate } from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { property } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { DropdownItemComponentAttributes } from './dropdown-item.model';
+import { styles } from './dropdown-item.styles';
+
+@hydrate()
+export class DropdownItemComponent
```

Comment:
Not sure about moving it to the UI. What is the aim of this component then? It is just an oryx-button with a bit of additional styles. Will we use it directly in EB compositions?

Reply:
I was trying to follow the suggestion above https://github.com/spryker/oryx/pull/708#discussion_r1325740926

and after discovering it wouldn't work because of a circular dependency with `ContentMixin`, we decided to just split the component like this. Would it have been better to keep it under the site package?

The only other minor difference is that we can use the ui dropdown-item component directly in login link instead of the site version (minor difference I guess).